backend/src/v5/processors/teamspaces/projects/models/containers.js

```diff
@@ -135,4 +136,19 @@ Containers.getSettings = (teamspace, container) => getContainerById(teamspace,
 
 Containers.getRevisionMD5Hash = getModelMD5Hash;
 
+Containers.getTree = async (teamspace, container, revision) => {
```

Comment:
Can we move this to `..../models/common/assets/jsonAssets.js`?

Just being wary of what's coming ahead we'll be better off putting this to a separate file.